src/Control/Lens/Internal/TH.hs

```diff
@@ -96,6 +96,14 @@ fromSet f x = Map.fromDistinctAscList [ (k,f k) | k <- Set.toAscList x ]
 newNames :: String {- ^ base name -} -> Int {- ^ count -} -> Q [Name]
 newNames base n = sequence [ newName (base++show i) | i <- [1..n] ]
 
+-- | Compatibility shim for pre-ConstraintKinds equality predicates.
+equalT :: Type -> Type -> Pred
```

Comment:
Don't do this. Use [`equalPred`](http://hackage.haskell.org/package/th-abstraction-0.2.3.0/docs/Language-Haskell-TH-Datatype.html#v:equalPred) from `th-abstraction` instead, which is fully backwards compatible.